dev/release/02-source.sh

```diff
@@ -25,6 +25,7 @@ set -eu
 : ${SOURCE_UPLOAD:=${SOURCE_DEFAULT}}
 : ${SOURCE_PR:=${SOURCE_DEFAULT}}
 : ${SOURCE_VOTE:=${SOURCE_DEFAULT}}
+: ${SOURCE_DOWNLOAD:=${SOURCE_DEFAULT}}
```

Comment:
Could you move this to between `SOURCE_DEFAULT` and `SOURCE_RAT`?

We use the same order as the real process order for these `SOURCE_*`.

```shell
: ${SOURCE_DEFAULT:=1}
: ${SOURCE_DOWNLOAD:=${SOURCE_DEFAULT}}
: ${SOURCE_RAT:=${SOURCE_DEFAULT}}
...

if [ ${SOURCE_DOWNLOAD} -gt 0 ]; then
  ...
fi

if [ ${SOURCE_RAT} -gt 0 ]; then
  ...
fi
...
```